docs/guides/server/management-interface.adoc

```diff
@@ -0,0 +1,51 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Configuring the Management Interface"
+summary="Learn how to configure {project_name}'s management interface for endpoints like metrics and health checks."
+includedOptions="http-management-* https-management-* legacy-observability-interface">
+
+The management interface allows accessing management endpoints via a different HTTP server than the primary one.
+It provides the possibility to hide endpoints like `/metrics` or `/health` from the outside world and, therefore, hardens the security.
+The most significant advantage might be seen in Kubernetes environments as the specific management port might not be exposed.
+
+== Management interface configuration
+
+The management interface is turned on by default, so management endpoints such as `/metrics`, and `/health` are exposed on the default management port `9000`.
+In order to change the port for the management interface, you can use the {project_name} option `http-management-port` or environment variable `KC_HTTP_MANAGEMENT_PORT`.
```

Comment:
Just a nitpick. I would not mention the env vars. IMHO various config sources and formats are out of scope for this guide. If we do that, we might as well talk about format for keystore etc.